data/json/items/armor/ballistic_armor.json

```diff
@@ -97,9 +102,11 @@
     "name": { "str": "ESAPI ballistic plate" },
     "description": "A polygonal ballistic plate with a slightly concave profile, made of ceramic.  Its inner surface is coated with UHMW polyethylene and is labeled \"TOP\", while the outer surface is labeled \"STRIKE FACE\".  This is intended to be worn in a ballistic vest, and can withstand several high-energy rifle rounds before breaking.",
     "weight": "2500 g",
-    "volume": "1533 ml",
+    "volume": "1715 ml",
+    "longest_side": "32 cm",
+    "//": "ESAPI Medium are 24cm x 31.75cm and 2.25cm thick",
```

Comment:
they are 25cm thick as shown in the armor entry bellow.

Reply:
25cm?  I think you meant 25mm here but correct me if I'm completely misunderstanding something here.